man/Makefile.am

```diff
@@ -1,5 +1,11 @@
 AUTOMAKE_OPTIONS = gnu
 
```

Comment:
@dachary yeah, that's what i had over my head when preparing this patch, but i was thinking about `make dist`.

actually i messed around the sphinx-build just try to force it to generate only the specified man page, but it seems that this idea needs more work. the list of man pages to be generated is created in `conf.py`. this script is basically something like `find . -name '*.rst'` in python. and yes, sphinx-build accept one or more files but the `man_pages` setting overrides the passed in argument. but if i do not set the `man_pages` in the `conf.py`, sphinx-build bails out compaining for failing to find the doctree!

was wondering that a better alternative was `rst2man`. but wheni ran this tool with some of our rst files for man page, it failed. i didn't look into it closer. since i think we should keep using a single document markup language and a single dialect in our source tree for ease of maintenance.